lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -145,9 +145,11 @@ func (h *StorageHandlersImpl) configureVolumeStores(op trace.Operation, handlerC
 			op.Error(err)
 		}
 
-		// if an error has been logged skip volume store cache addition
+		// if an error has been logged, stop starting port-layer which would cause port-layer-server to restart continually
 		if err != nil {
-			continue
+			// continue
+			op.Errorf("%s:%s volume store error %s", dsurl.String(), name, err)
+			os.Exit(0)
```

Comment:
Exit(1)

Reply:
Agree with Jun.

Reply:
@wjun @yuyangbj  If any error from both nfs-datastore and vs-datastore occurs, the portlayer would exit. Is it OK, or just exit only if nfs-datastore error occurs?